### PR TITLE
Get document number from PACER Download confirmation page

### PR DESCRIPTION
**Pull request by albertisfu**
Added a new class `DownloadConfirmationPage` for querying and parsing the download confirmation page and retrieving the document number from this page.

This has support for querying appellate download confirmation pages and getting the document number for all appellate jurisdictions even though we're only going to use this for `ca8`, `ca11`, and `cadc` documents or in case we cannot get the document number from the PDF from other courts.

### CODE REVIEW

File: juriscraper/pacer/download_confirmation_page.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+        # Make the NDA document URL
+        url = make_docs1_url(self.court_id, pacer_doc_id)
+
+        logger.info("Querying the confirmation page endpoint at URL: %s", url)
+        self.response = self.session.get(url)
+        self.parse()
+
+    @property
+    def data(self):
```
Comment (mlissner) [inline juriscraper/pacer/download_confirmation_page.py:40]: It feels like if we're going to have an object for this page, we should gather everything from it that we can. That'd probably include:

 - docket_number
 - cost
 - billable_pages
 - document_description
 - transaction_date

I assume that wouldn't be too hard, and it's probably worth doing, just so things are complete if anybody needs this class for something else. Nothing is more annoying than realizing that a tool is only half done.

**Review by mlissner — CHANGES_REQUESTED** [review]
This looks like a good architecture, but a couple small comments. One will slow you down, the other will be easy.

File: juriscraper/pacer/download_confirmation_page.py (hunk 1 above)
Comment (albertisfu) [reply juriscraper/pacer/download_confirmation_page.py:40]: Great, yeah I was not sure what other data might be useful for this object. Now we get this data.

**Comment by albertisfu** [comment]
Thank you! I've applied the changes, adding the new data to the report.
Let me know if we need something else here.

**Pull request merged**